docs/active-directory/core-concepts/01-active-directory-details-and-definitions.md

```diff
@@ -0,0 +1,111 @@
+# Active Directory Details and Definitions 
+
+### Active Directory Domain Services
+
+Active Directory Domain Services (AD DS, or more simply just AD for Active Directory) is a storage and management
+mechanism for everything related to a collection of resources within a computer network. This includes users, files,
+network configurations, and devices like physical computers and printers. Administrators can modify the Active Directory
+and remotely manage all the components currently tracked at once. In its simplest terms, **AD is used to logically
+represent an organizations resources and users and enforce policies and behaviors across them.**
```

Comment:
```suggestion
Active Directory Domain Services (often informally referred to as AD DS or AD) is a storage and management mechanism for everything related to a collection of resources within a computer network, including users, files, network configurations, and devices like physical computers and printers. 

In its simplest terms, Active Directory is used to **create a logical representation of an organization's resources and users**, which allows them to **remotely manage these resources** and **enforce policies and behaviors across them**.
```

Reply:
in general, there seem to be manually added newlines! Instead, I think a better approach is to create line breaks between sentences (instead of defining paragraphs) and let sentences stay on the same line.